brightener.h

```diff
@@ -2,11 +2,6 @@
 #include "image.h"
 
 class ImageBrightener {
-private:
-	std::shared_ptr<Image> m_inputImage;
 public:
-	ImageBrightener(std::shared_ptr<Image> inputImage);
-	bool ValidateImage();
-	int BrightenWholeImage();
-	bool AddBrighteningImage(Image& imageToAdd, int& attenuatedCount);
+	static int BrightenWholeImage(std::shared_ptr<Image> image);
```

Comment:
Good approach to remove unwanted functions and make this a static function.
Is it necessary to have it as a `class` or is it more like a namespace with a function in it?

Reply:
Free floating function yes, was torn between class and namespace :-)